nomad/structs/structs.go

```diff
@@ -1146,7 +1146,7 @@ func (sc *ServiceCheck) Hash(serviceId string) string {
 }
 
 const (
-	NomadConsulPrefix = "nomad"
+	NomadConsulPrefix = "1729nomad"
```

Comment:
or n0m4d ;D Honestly, that's a minor detail to me, as long as it does not easily clash with users's service IDs I'm ok with whatever prefix.